lib/data/multipleBackendGateway.js

```diff
@@ -221,6 +221,22 @@ const multipleBackendGateway = {
         }
         return cb();
     },
+
+    copyObject: (request, locationType, objectGetInfo, log, cb) => {
+        const client = clients[locationType];
+        if (client.copyObject) {
+            return client.copyObject(request, locationType, objectGetInfo,
+            log, (err, key) => {
+                const dataRetrievalInfo = {
+                    key,
+                    dataStoreName: locationType,
+                    dataStoreType: client.clientType,
+                };
+                cb(err, dataRetrievalInfo);
+            });
+        }
+        return cb();
```

Comment:
So if the client is not AWS and a copy object request is made, do we handle that?
`process.nextTick` ?

Reply:
I added a note too.
We use `copyObject` only if copying object from one external backend to the same external backend

Reply:
So could someone make a request to copy Azure object to another location in Azure?
Just wondering if we need to return NotImplemented or not -- assuming we don't have a `copyObject` for the Azure backend client yet

Reply:
not yet, so I added a NotImplemented error, thanks :)